.travis.yml

```diff
@@ -33,5 +34,8 @@ notifications:
     on_failure: change
 env:
   - secure: "OmaaZ3jhU9VQ/0SYpenUJEfnmKy/MwExkefFRpDbkRSu/hTQpxxALAZV5WEHo7gxLRMRI0pytLo7w+lAd2FlX1CNcyY62MUicta/8P2twsxp+lR3v1bJ7dwk6qsDbO7Nvv3BKPCDQCHUkggbAEJaHEQGdLk4ursNEB1aGimuCEc="
+  matrix:
```

Comment:
I think this either needs a `-` in front, or the above `- secure:` needs to become `secure:`